### ISSUE TITLE
Possibility to send multiple messages at once

### ISSUE THREAD

**Issue by carloscanom** [Ankit404butfound/PyWhatKit#124]
How about making a function called sendMultipleMsgs which loops through a list of messages to be sent?

Something like ["Hello", "How are you??", "Everything fine??"] sent as three different messages one at a time.

**Comment by aaryanrr** [comment]
Hey @carloscanom, thanks for the great idea. We'll implement this in Version 5.3.

**Comment by codewithpom** [comment]
Yes why not simply do it and make a PR and then I will simply check it.

**Comment by Tanish-Panwar** [comment]
I want to work with this issue.

**Comment by codewithpom** [comment]
@Tanish-Panwar thanks for your support and pleae4 so it as it is a very easy function I will select that whose code is more efficient and good for our module so simply fork it and then make the changes and then make a PR.
